release_build_files/readme.md

```diff
@@ -573,6 +573,10 @@ code.
     -   General: Variant double type now support 64-bit while saving to json.
         ([#1133](https://github.com/firebase/quickstart-unity/issues/1133)).
     -   Analytics (tvOS): Analytics is now supported on tvOS.
+    -   Firestore: Graduated from "beta" to "generally available".
```

Comment:
"Graduated" seems a little cutesy, what about just "Firestore: Firestore is now released to general availability (no longer Beta)." or rephrased without the parentheses - also I'd be fine with making this first bullet point.

Reply:
How's this: Firestore: Released to general availability (no longer beta).